docs/features/feature-guides/compliance-forms/complete-a-form.md

```diff
@@ -0,0 +1,176 @@
+       alt="Open Compliance Form from Task Center"/>
+</p>
+
+## Completing a Form for a Single Asset
+
+When filling out a Compliance Form for a single Asset, you'll see a list of Questions tailored to that Asset, with clear labels showing which ones are required. Here's how it works:
+
+- **Question Details:** Each Question specifies if it's required or optional. Required Questions must be completed to submit the Form.
+- **Pre-Populated Metadata:** If metadata already exists for a Question, it will appear pre-filled. You can confirm the existing value or make updates as needed.
+- **Assignee Contributions:** If another Assignee has already provided a response, their name and the time of submission will be displayed. This gives you visibility into previous input, though you can still update the response.
+
+:::tip
+For Verification Forms, after addressing all required Questions, you'll be prompted to provide final sign-off. This ensures all responses are complete and accurate, marking the Form ready for submission.
+:::
+
+Once you complete all required responses, the sidebar will update with the status of the Asset:
+
+- **Documented**: All required Questions are completed, Verification is not needed
+- **Verified**: All required Questions are completed and Verified
+
+Here's what the **Governance Initiative 2024** Verification Form looks like for `dogs_in_movies` after responding to all Required Questions:
+
+<p align="center">
+  <img
+       width="80%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/completion/complete-task-ready-to-verify.png"
+       alt="Asset Ready to Verify"/>
+</p>
+
+And here's the `dogs_in_movies` sidebar after Verifying all responses:
+
+<p align="center">
+  <img
+       width="70%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/completion/complete-task-asset-verified.png"
+       alt="Asset is Verified"/>
+</p>
+
+### Navigating to the Next Asset
+
+To continue working through the Compliance Forms assigned to you, **use the navigation arrows located in the top-right corner**. These arrows will take you to the next Asset that is still pending Form completion or Verification. Only Assets that require action will appear in this flow, allowing you to focus on the remaining tasks without unnecessary steps.
+
+
+## Completing a Form Question for Multiple Assets
+
+When you want to provide the same response for a question to multiple assets, you can apply it in bulk by selecting the **By Question** option in the top-right corner. This allows you to navigate through the Form question-by-question and apply the same response to multiple assets.
+
+:::note
+Completing Form Questions for multiple Assets is only supported for DataHub Cloud.
+:::
+
+### Example: Applying a Response in Bulk
```

Comment:
This is nice!

Reply:
I would recommend a brief introduction sentence, though.

Imagine we are trying to provide the same answer to a question for all assets in a Snowflake schema called `DEMO_DB.` Here's how we'd do it: